test/numbers.coffee

```diff
@@ -66,12 +66,15 @@ test "Python-style octal literal notation '0o777'", ->
   eq Number::toString, 0o777['toString']
   eq Number::toString, 0o777.toString
 
-test "#2060: Disallow uppercase radix prefixes and exponential notation", ->
-  for char in ['b', 'o', 'x', 'e']
+test "#2060: Disallow uppercase radix prefixes", ->
+  for char in ['b', 'o', 'x']
     program = "0#{char}0"
     doesNotThrow -> CoffeeScript.compile program, bare: yes
     throws -> CoffeeScript.compile program.toUpperCase(), bare: yes
 
+test "#5164: Allow uppercase exponent notation", ->
+  doesNotThrow -> CoffeeScript.compile "0E0", bare: yes
```

Comment:
Can we add a corresponding test that the lowercase version is also still permitted?

Reply:
Done